components/studio/libexec/hab-studio-type-default.sh

```diff
@@ -6,6 +6,7 @@ studio_build_environment=
 studio_build_command="record \${1:-} $HAB_ROOT_PATH/bin/build"
 studio_run_environment=
 studio_run_command="$HAB_ROOT_PATH/bin/hab pkg exec core/hab-backline bash -l"
+studio_supervisor_start_command="$HAB_ROOT_PATH/bin/hab sup run > /hab/sup/log.out &"
```

Comment:
You should place this in `/hab/sup/default/out.log` - the supervisor that we're starting is "named" default. If somebody starts a separate supervisor they might want to consider outputting that to `/hab/sup/<name>/out.log`